src/assets/scripts/client/parsers/scopeCommandParser/ScopeCommandModel.js

```diff
@@ -0,0 +1,66 @@
+import _first from 'lodash/first';
+import _has from 'lodash/has';
+import _last from 'lodash/last';
+import _without from 'lodash/without';
+import _values from 'lodash/values';
+import {
+    IMPLIED_COMMANDS,
+    EXPLICIT_COMMANDS,
+    COMMAND_FUNCTIONS
+} from './scopeCommandMap';
+
+// FIXME: Remove dummy sector codes
```

Comment:
I 👀 a `FIXME`

Reply:
Yeeahhh I guess that'll have to be a todo anyway. Involves also hashing out the SectorModel and SectorCollection.